src/Microsoft.AspNet.Mvc.ViewFeatures/Controller.cs

```diff
@@ -102,21 +102,24 @@ public ModelStateDictionary ModelState
         }
 
         /// <summary>
-        /// Gets or sets the <see cref="Mvc.ActionContext"/> object.
+        /// Gets or sets the <see cref="Mvc.ControllerContext"/>.
         /// </summary>
         /// <remarks>
         /// <see cref="Controllers.IControllerActivator"/> activates this property while activating controllers.
         ///  If user code directly instantiates a controller, the getter returns an empty
-        /// <see cref="Mvc.ActionContext"/>.
+        /// <see cref="Mvc.ControllerContext"/>.
         /// </remarks>
-        [ActionContext]
-        public ActionContext ActionContext
+        [ControllerContext]
+        public ControllerContext ControllerContext
```

Comment:
The main impact of this change is to unit tests, or where someone wrote unnecessarily verbose code like `ActionContext.HttpContext`.